packages/front-end/components/Features/FeaturesOverview.tsx

```diff
@@ -1073,6 +1082,18 @@ export default function FeaturesOverview({
                         Default Value.
                       </p>
                     )}
+                    {!archetypes.length ? (
```

Comment:
This is only relevant when you have a complex feature flag.  Only show if there are 2+ rules.  Because of that, I'm wondering if this makes sense beneath the environment tabs instead of above them.